notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Plot generator model.
</pre><p>Should be "Plot discriminator model."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>